### ISSUE TITLE
Manual Partition Bug

### ISSUE THREAD

**Issue by brando56894** [deadhead420/arch-linux-anywhere#16]
Whenever I choose to manually partition my drive it immediately displays a warning that says the selected partition is too small to be a root partition. Once I hit "OK" it pops up again, the only way to get out of it is to switch to another console and kill the process.

**Comment by OscarHolst** [comment]
@brando56894 i will look into it.

**Comment by jscholfield** [comment]
The code looks like it is requiring a partition greater than 4 GB for root and that must be done first. Were you using a 4 GB or less partition?

**Comment by OscarHolst** [comment]
@jscholfield i haven't got the time to look into @brando56894's issue, but if i understood him correctly he get's that error before he even got the time to partition the drive.

**Comment by OscarHolst** [comment]
@brando56894 i have tried a couple of times now and i only encounter your "problem" when i use less than 4GB for root as @jscholfield already stated and that is a "fail-safe" behavior . How ever, i do not get stuck in the process as you explain and i could easily navigate back in the menu. I sincerely hope by now that you have successfully re-sized the partition and installed Arch Linux on your computer.

How ever, if the problem persists or you encounter any other trouble along the way, don't hesitate to write back.

**Comment by deadhead420** [comment]
I will try as well to recreate this issue. So far I haven't been able to make it happen in any VM. Will test on real hardware when I get home... But so far have not been able to trap myself in any menus.

**Comment by deadhead420** [comment]
I figured it out. The bug only happens when using efi boot. The way I have it setup lsblk runs and displays all the device info in the partition menus. I was excluding a match with grep so that the install media is not shown in the partition menus. Turns out that broke the lsblk command when the iso is booted in efi mode and no devices are shown causing the whiptail command to fail.

I have already come up with a fix and am testing it now. This bug will not be present in the next release 2.1.5 to be released in a day or two. Thanks for pointing out this major bug as it would break the installer for anyone using efi. Check my website for the release of 2.1.5 soon containing the patch for this issue. Closing the issue thanks again.
